### PR TITLE
 added first concept

### PR DESCRIPTION
**Pull request by EugenWiens**
I added the first concept of the **head**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 # jenkins-head
 This repository contains gatt-server and gatt-client software. Main goal is to visualize the Jenkins state :) 
 
-# The GATT-server is based on the project of Douglas Otwell
+##The GATT-server is based on the project of Douglas Otwell
```
Comment (leinher) [inline README.md]: This belongs to section `GATT-Server`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- jobName: "job2"
+  jobUrl: "http://..."
+```
+### GATT characteristics
+TBD
+
+## GATT Client
+The GATT Client is colouring the jenkins head regarding the state. 
+- red => There are errors
+- green => Every thing is OK
```
Comment (leinher) [inline README.md]: If a person has a green head he/she is probably sick. So the head should be white if everything is OK.

To keep it as simple as possible the head should just read the **RGB** values from the GATT Server.
It should not know the different states.

**Comment by EugenWiens** [comment]
> It should be possible to server multiple heads from one GATT server.

yes, your are right. What about that the GATT server creates one BLE service per a watched job.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+|             |    REST    |              |     BLE    | |              |  |
++-------------+    API     +--------------+            | +--------------+  |
+                                                       |      Jenkins      |
+                                                       |       Head        |
+                                                       |                   |
+                                                       +-------------------+
+```
+
+## GATT Client
+The GATT Client collects the state of all registered Jenkins jobs. After that the server joins all states to one global state. This global state is represented as an **RGB** value is pushed by the GATT Client to the GATT Server (Jenkins Head)
```
Comment (leinher) [inline README.md]: Dot missing

File: README.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
+
+### Jenkins jobs
+The jobs that should be monitored are represented as an entry in a YAML file e.g.
+```yaml
+- client: "client Name (jenkins head1)"
```
Comment (leinher) [inline README.md]: I think we should name that `clientName`

**Pull request merged**